### PR TITLE
helm: Add replace parameter

### PR DESCRIPTION
**Pull request by Akasurde**
##### SUMMARY

Fixes: #106

Signed-off-by: Abhijeet Kasurde <akasurde@redhat.com>

##### ISSUE TYPE
- Bugfix Pull Request

##### COMPONENT NAME
changelogs/fragments/106-helm_replace.yml
molecule/default/roles/helm/tasks/tests_chart.yml
plugins/modules/helm.py

### CODE REVIEW

**Comment by Akasurde** [comment]
@LucasBoisserie Could you please review this? Thanks.

File: plugins/modules/helm.py [hunk 1]
```diff
@@ -455,6 +473,10 @@ def main():
     # keep helm_cmd_common for get_release_status in module_exit_json
     helm_cmd = helm_cmd_common
     if release_state == "absent" and release_status is not None:
+
+        if not replace and replace:
```
Comment (fabianvf) [inline plugins/modules/helm.py:477]: I think one of these variables is probably supposed to be different

**Review by fabianvf — COMMENTED** [review]
one issue I spotted, otherwise looks great

File: plugins/modules/helm.py (hunk 1 above)
Comment (Akasurde) [reply plugins/modules/helm.py:477]: oh My bad. I will update.

**Pull request merged**